yb-voyager/cmd/metadataDB.go

```diff
@@ -97,3 +97,35 @@ func truncateTablesInMetaDb(exportDir string, tableNames []string) error {
 	}
 	return nil
 }
+
+// =====================================================================================================================
+
+type MetaDB struct {
+	db *sql.DB
+}
+
+func NewMetaDB(exportDir string) (*MetaDB, error) {
+	db, err := sql.Open("sqlite3", getMetaDBPath(exportDir))
+	if err != nil {
+		return nil, fmt.Errorf("error while opening meta db :%w", err)
+	}
+	return &MetaDB{db: db}, nil
+}
+
+func (m *MetaDB) MarkSegmentAsProcessed(segmentNum int64) error {
```

Comment:
How about `MarkEventQueueSegmentAsProcessed` instead just to make it more explicit?

Reply:
Done